src/js/components/Layout/Breadcrumbs.jsx

```diff
@@ -1,11 +1,70 @@
-import React from 'react';
+/* eslint-disable react/no-array-index-key */
+import React, { Component } from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { showModal } from '../../actions';
 
-const Breadcrumbs = () => (
-  <div className="bradcrumbs-container d-flex">
-    <span>
-      Breadcrumbs
-    </span>
-  </div>
-);
+class Breadcrumbs extends Component {
+  constructor(props) {
+    super(props);
 
-export default Breadcrumbs;
+    this.openModal = this.openModal.bind(this);
+  }
+
+  componentDidMount() {
+
+  }
+
+  openModal() {
+    this.props.showModal();
+  }
+
+  render() {
+    const listItems = window.location.pathname.split('/');
+    const listItemFormated = [];
+    _.forEach(listItems, (name) => {
+      // Put a space before each uppercase
+      let nameFormated = name.replace(/([A-Z])/g, ' $1').trim();
+      // Put an uppercase at the beginning of each word
+      nameFormated = nameFormated.replace(/(^\w{1})|(\s{1}\w{1})/g, match => match.toUpperCase());
+      listItemFormated.push(nameFormated);
+    });
+    const listToReturn = listItemFormated.map((name, id) =>
+      (name === 'Openboxes' || name === '' ? null : (
+        <a key={`item-${id}`} href={window.location.pathname.split('/', id + 1).join('/')} className="item-breadcrumbs"> {name}
```

Comment:
Move `{name}` to the next line for better readability (here it goes out of the screen and was misleading for me)